src/components/table/DataTable.js

```diff
@@ -0,0 +1,24 @@
+import { default as ReactDataTable } from 'react-data-table-component'
+import styled from 'styled-components/macro'
+
+const DataTableWrapper = styled(ReactDataTable)`
+  .rdt {
+    &_TableHeadRow {
+      font-style: normal;
+      font-weight: bold;
+      font-size: 14px;
+    }
+    &_TableCell {
+      font-size: 14px;
+      &[data-column-id='scientific_name'] {
+        font-style: italic;
+      }
+    }
+  }
+`
+
+const DataTable = ({ data, columns }) => (
```

Comment:
You can just take `(props)` here and do `<DataTableWrapper pagination {...props} />` so that you can take all of the props referenced here: https://react-data-table-component.netlify.app/?path=/docs/api-props--page

Reply:
➕ 1️⃣